src/components/principal/unit/buttons/UnitButtons.vue

```diff
@@ -6,11 +6,11 @@
         :disabled="isShiftExchangeNotAllowed"
         @click="shiftExchange()"
       >
-        <span class="button-font-size">Schimb de tură</span>
+        <span class="button-font-size">{{shiftExchangeButtonText}}</span>
       </button>
     </div>
     <div>
-      <button class="custom-button btn" :disabled="isUnitLocked" @click="addResources()">
+      <button class="custom-button btn" v-show="userHasRightsToEditCurrentUnit" :disabled="isUnitLocked" @click="addResources()">
```

Comment:
Is it always the role 'supervisor'?  is it case insensitive?

Reply:
> Is it always the role 'supervisor'? is it case insensitive?

It is how we specify it from keycloak, which at the moment should be supervisor, but we can change that as we wish. However this raises the point which i've missed: supervisor role should always have access to all modules (services, reports, etc) at at the moment it doesn't have that.